executors/cpp/list_fmt.cpp

```diff
@@ -113,7 +113,7 @@ string TestListFmt (json_object* json_in) {
     return json_object_to_json_string(return_json);
   }
 
-  UnicodeString *u_array = &u_strings[0];
+  UnicodeString *u_array = u_strings.data();
```

Comment:
This is a strange suggestion from Clang. I think you want [`vector::front`](https://cplusplus.com/reference/vector/vector/front/)?

Reply:
No, .front() gives the first element's value, not its location. In this case .data() effectively gives a pointer to the start of the array.

Reply:
https://cplusplus.com/reference/vector/vector/data/

Reply:
Okay, is see the difference in the data type of the returned value between the two functions. And yeah, `.data()` returns a pointer to the internal array, and C++ guarantees the array of a vector is entirely contiguous in memory, so this works.